apps/condo/domains/common/utils/codegeneration/generate.gql.js

```diff
@@ -85,7 +105,9 @@ function generateGqlQueries (key, fields) {
         GET_COUNT_OBJS_QUERY,
         GET_ALL_OBJS_WITH_COUNT_QUERY,
         CREATE_OBJ_MUTATION,
+        CREATE_OBJS_MUTATION,
         UPDATE_OBJ_MUTATION,
+        UPDATE_OBJS_MUTATION,
```

Comment:
delete many? %)

Reply:
Why do you even need this?)))0)

Reply:
Since all deletions are restricted

Reply:
We would need a way to explicitly delete everything that have deletedAt field at some point. It would be safer to do it in condo. And it would be faster to do it in bulk mode

Reply:
I've added utils + tests